docs/src/content/docs/it/environmental-impact.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Documentazione ecologica
+description: Impara come Starlight può aiutarti a costruire siti per documentazione più verdi e ridurre la tua impronta ecologica.
+---
+
+Le stime dell'impatto ambientale del web si aggirano tra il [2%][sf] e il [4% delle emissioni globali di gas serra][bbc], equivalente circa alle emissioni dell'industria aerea.
+Ci sono molti complessi fattori per calcolare l'impatto ambientale di un sito web, ma questa guida include dei consigli per ridurre l'impronta ecologica del tuo sito.
+
+La buona notizia è che scegliere Starlight è un ottimo punto di partenza.
+Secondo il Website Carbon Calculator, questo sito è [più pulito del 99 % delle pagine web testate][sl-carbon], producendo 0,01 g di CO₂ per pagina visitata.
```

Comment:
Is "%" separated from the number in Italian? If not, here's a "one click fix"!

```suggestion
Secondo il Website Carbon Calculator, questo sito è [più pulito del 99% delle pagine web testate][sl-carbon], producendo 0,01 g di CO₂ per pagina visitata.
```

Reply:
Hi, thank you for noticing that. 
You're right, in Italian it shouldn't be separated.